.circleci/config.yml

```diff
@@ -175,21 +219,28 @@ jobs:
 
   test-11-12_check-pg-upgrade:
     docker:
-      - image: 'citus/pgupgradetester:latest'
+      - image: 'heisenberg302/pgupgradetester:latest'
     working_directory: /home/circleci/project
     steps:
+      - checkout
       - attach_workspace:
           at: .
+      - run:
+          name : "install necessary tools"
+          command: |
+              apt-get update
+              apt-get install -y autoconf build-essential libcurl4-openssl-dev libicu-dev libreadline-dev libselinux1-dev libxslt-dev libssl-dev
```

Comment:
these are in the image right? if not, lets add to the image

Reply:
Yeah, this is also a leftover.